app/models/identity.rb

```diff
@@ -84,8 +88,9 @@ class Identity < ApplicationRecord
 
   validates :ldap_uid, uniqueness: {case_sensitive: false}, presence: true
   validates :orcid, format: { with: /\A([0-9]{4}-){3}[0-9]{3}[0-9X]\z/ }, allow_blank: true
+  # personal demographics are only required on Edit Profile page
+  validate :validate_demographics, if: Proc.new { |i| i.id == i.updater_id && i.sign_in_count > 0 }
 
-  # validates_presence_of :reason, if: :new_record?
```

Comment:
Didn't notice this one earlier.  Was this also removed on purpose?

Reply:
yes, it was. It was already commented out, so I deleted it. I will add it back for the record.